smarts/env/hiway_env.py

```diff
@@ -129,22 +129,59 @@ def __init__(
         if visdom:
             visdom_client = VisdomClient()
 
-        self._smarts = SMARTS(
-            agent_interfaces=agent_interfaces,
-            traffic_sim=SumoTrafficSimulation(
+        is_opendrive = HiWayEnv.check_scenario_versions(scenarios)
+        traffic_sim = None
+        zoo_addrs = None
+        if is_opendrive:
+            # We currently don't support the Native SUMO Traffic Provider or Social Agents for OpenDRIVE maps
+            pass
+        else:
+            from smarts.core.sumo_traffic_simulation import SumoTrafficSimulation
+            traffic_sim = SumoTrafficSimulation(
                 headless=sumo_headless,
                 time_resolution=fixed_timestep_sec,
                 num_external_sumo_clients=num_external_sumo_clients,
                 sumo_port=sumo_port,
                 auto_start=sumo_auto_start,
                 endless_traffic=endless_traffic,
-            ),
+            )
+            zoo_addrs = zoo_addrs
```

Comment:
Sorry, I was still in the middle of editing.

```suggestion
```

Reply:
oh sorry, about that! But i have included the above change.